stdlib/2and3/ctypes/__init__.pyi

```diff
@@ -49,9 +49,10 @@ class _CDataMeta(type):
     # TODO The return type is not accurate. The method definition *should* look like this:
     # def __mul__(cls: Type[_CT], other: int) -> Type[Array[_CT]]: ...
     # but that is not valid, because technically a _CDataMeta might not be a Type[_CT].
-    # This can never actually happen, because all _CDataMeta instances are _CData subclasses, but a typechecker doesn't know that.
-    def __mul__(cls: _CDataMeta, other: int) -> Type[Array[_CT]]: ...
-    def __rmul__(cls: _CDataMeta, other: int) -> Type[Array[_CT]]: ...
+    # This can never actually happen, because all _CDataMeta instances are _CData subclasses,
+    # but a typechecker doesn't know that.
+    def __mul__(cls: _CDataMeta, other: int) -> Type[Array[Any]]: ...
```

Comment:
So why don't you want to write
```python
def __mul__(cls: Type[_CT], other: int) -> Type[Array[_CT]]: ...  # type: ignore
```
It looks like we are loosing lots of precise types in user code from some "perfectionist" considerations of not wanting to write `# type: ignore`.

Reply:
You're completely right. I'm not sure what happened, I thought I made that change, but apparently I never committed it. Will fix that.